source/release-notes/1.1.txt

```diff
@@ -0,0 +1,44 @@
+.. _c2c-release-notes-1.1:
+
+===================================
+Release Notes for ``mongosync`` 1.1
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   ``mongosync`` 1.1 Released January 13, 2023
+
+
+Filtered Sync
+-------------
+
+{+c2c-product-name+} provides continuous data synchronization or a 
+one-time data migration between two MongoDB clusters. You can use
+filtered sync to control the data that the :ref:`mongosync
+<c2c-mongosync>` utility transfers between the source and destination
+clusters.
+
+For more information, see: :ref:`c2c-filtered-sync`.
+
+Unlike Sharded Topologies
+-------------------------
+
+{+c2c-product-name+} 1.1 supports sync between clusters with different
```

Comment:
I feel this description might be 100% accurate. @alan-zheng-mdb You might be able to help with the wording here?

Reply:
@huan-Mongo  That's the wording from the Filtered Sync feature page.

Reply:
I think we may just want to clarify that you can go SC -> SC or RS -> SC, but not SC -> RS.

Reply:
Done

Reply:
Yes - I rather be more prescriptive that you can go Replica Set to Sharded Cluster.

We also don't mention that we can do M shard to N Shard. I understand we don't need special parameters, but mentioning here that this capability is enabled in 1.1 is a good reference.

Reply:
Done